PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-vm/src/main/java/com/microsoft/azure/toolkit/intellij/vm/ssh/BrowseRemoteHostSftpAction.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.vm.ssh;
+
+import com.intellij.openapi.project.Project;
+
+import com.intellij.openapi.util.Pair;
+import com.intellij.openapi.wm.ToolWindow;
+import com.intellij.remote.AuthType;
+import com.intellij.ssh.config.unified.SshConfig;
+import com.intellij.ui.content.Content;
+import com.jetbrains.plugins.webDeployment.config.AccessType;
+import com.jetbrains.plugins.webDeployment.config.GroupedServersConfigManager;
+import com.jetbrains.plugins.webDeployment.config.WebServerConfig;
+import com.jetbrains.plugins.webDeployment.config.WebServerGroupingWrap;
+import com.jetbrains.plugins.webDeployment.ui.WebServerToolWindowFactory;
+import com.jetbrains.plugins.webDeployment.ui.WebServerToolWindowPanel;
+import com.microsoft.azure.toolkit.lib.common.messager.AzureMessager;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.microsoft.azure.toolkit.lib.compute.virtualmachine.VirtualMachine;
+import org.apache.commons.lang3.reflect.MethodUtils;
+
+import javax.annotation.Nonnull;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Objects;
+
+
+public class BrowseRemoteHostSftpAction {
+
+    @AzureOperation(name = "vm.browse_files_sftp", params = "vm.getName()", type = AzureOperation.Type.ACTION)
+    public static void browseRemoteHost(VirtualMachine vm, @Nonnull Project project) {
```

Comment:
[DONE] split to 3 steps
1. get ssh config
2. get webserver config from ssh
3. open remote host in toolwindows